Lib/python/pyhead.swg

```diff
@@ -93,6 +93,29 @@ SWIG_Python_str_FromChar(const char *c)
 #define Py_hash_t long
 #endif
 
+// gh-106004 added PyDict_GetItemRef() and PyDict_GetItemStringRef()
```

Comment:
copy-paste from https://github.com/python/pythoncapi-compat/blob/main/pythoncapi_compat.h

Reply:
This was modified to work around a CI failure in `python 3.8 abi=3.5 gcc`:
```
[11:14:40] 💁 _example.abi3.so: 1 extensions scanned; 1 ABI version mismatches  
           and 0 ABI violations found                                           
make[3]: *** [../../Makefile:1319: python] Error 1
{"specs": {"_example.abi3.so": {"kind": "object", "object": {"name": "_example.abi3.so", "result": {"is_abi3": true, "is_abi3_baseline_compatible": false, "baseline": "3.5", "computed": "3.13", "non_abi3_symbols": [], "future_abi3_objects": {"PyDict_GetItemRef": "3.13", "PyDict_GetItemStringRef": "3.13"}}}}}}
```